code/core_init.c

```diff
@@ -76,9 +76,9 @@ void set_units(void)
 void read_snap_list(void)
 {
   FILE *fd;
-  char fname[1000];
+  char fname[MAX_STRING_LEN];
 
-  sprintf(fname, "%s", FileWithSnapList);
+  snprintf(fname, MAX_STRING_LEN, "%s", FileWithSnapList);
```

Comment:
I made this comment further down - but would be good to either allocate with a ` MAX_STRING_LEN + 1` or pass `MAX_STRING_LEN - 1` to `snprintf`